qgis_deployment_toolbelt/constants.py

```diff
@@ -225,7 +228,7 @@ def valid_shortcut_name(self, shortcut_name: str) -> bool:
     @classmethod
     def from_opersys(
         cls,
-        operating_system_codename: Literal["darwin", "linux", "win32"] | None = None,
+        operating_system_codename: str | None = None,
```

Comment:
Why `opersys` is not set as default value ? Just for the log ?

Reply:
To anticipate over future operating system whom code name is not within opersys module.